src/admin/bundles/bundles.factory.js

```diff
@@ -18,6 +18,55 @@
                 details: {method: 'GET', params: {action: 'detail'}}
             }
         );
+
+        Bundle.prototype.getVersion = function(){
+            return this.version.major+"."+this.version.minor+"."+this.version.micro+"-"+this.version.qualifier;
+        };
+        Bundle.prototype.getIconURL = function(){
+            return ServerService.formatURL("module/server/module/icon?bundleId=" + this.bundleId);
+        };
+        Bundle.prototype.isActive = function(){
+            return this.state === "ACTIVE";
+        };
+
+
+        function callbackSuccess(){
+            LoadingModal.close();
+            $rootScope.$broadcast('motech.refresh');
+        }
+
+        Bundle.prototype.start = function(){
+            LoadingModal.open();
+            this.$start(callbackSuccess);
+        };
+        Bundle.prototype.stop = function(){
+            MotechConfirm(i18nService.getMessage('admin.bundles.stateChangeWarning')) //'admin.bundles.stop' (stop button text)
+            .then(function(){
+                LoadingModal.open();
+                this.$stop(callbackSuccess);
+            });
+        };
+        Bundle.prototype.restart = function(){
+            LoadingModal.open();
+            this.$restart(callbackSuccess);
+        };
+        
+        Bundle.prototype.uninstallBundle = function(){
+            MotechConfirm(i18nService.getMessage('admin.bundles.stateChangeWarning')) // 'admin.remove' (remove button text)
```

Comment:
They really should be different, but in the original UI they are the same, and both read something like "changing a bundle will break MOTECH" -- which isn't really meaningful beyond having the user stop and check what they are doing.

I think there are two options maybe...
(1) Have 'start bundle' show the same message (because its true that starting a bundle could blow something up)
(2) Create a new start/stop/uninstall messages, and we can add specific translations, but these messages might be very similar but could be more descriptive...

I think option (1) is best... @jredlarski do you have an opinion?